modules/ocf_desktop/files/xsession/print-notify-handler

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+import functools
+import os
+from configparser import ConfigParser
+
+import redis
+
+
+BROKER_AUTH = '/opt/share/broker/broker.conf'
+
+redis_connection = functools.partial(
+    redis.StrictRedis,
+    host='broker',
+    port=6378,
+    ssl=True,
+    db=0,
+    password=None
+)
+
+
+def subscribe(host, password, *channels):
+    rc = redis_connection(host=host, password=password)
+    sub = rc.pubsub(ignore_subscribe_messages=True)
+    sub.subscribe(channel for channel in channels)
+    return sub
+
+
+def read_config():
+    config = ConfigParser()
+    config.read(BROKER_AUTH)
+    host = config.get('broker', 'host')
+    password = config.get('broker', 'password')
+    return host, password
+
+
+def main():
+    username = os.environ.get('SUDO_USER')
+
+    if not username:
+        raise RuntimeError('Unable to read SUDO_USER.')
```

Comment:
I'd report this [via ocflib](https://github.com/ocf/ocflib/blob/292f0d835d20f55512380945ecacd73f0b25a5fd/ocflib/misc/mail.py#L88) since otherwise I don't think we'll ever see it.